plugin/api.go

```diff
@@ -37,6 +37,16 @@ type API interface {
 	// Minimum server version: 5.2
 	UnregisterCommand(teamId, trigger string) error
 
+	// RegisterPluginIntegration registers a custom integration from the plugin.
+	//
+	// Minimum server version: 5.28
+	RegisterPluginIntegration(trigger *model.PluginIntegration) error
+
+	// UnregisterPluginIntegration unregisters a custom integration previously registered via RegisterPluginIntegration.
+	//
+	// Minimum server version: 5.28
+	UnregisterPluginIntegration(location, trigger string) error
```

Comment:
Is this supposed to be `requestURL` as a parameter? (The word `trigger` is unexpected in a few places in the proposed changes.)

Reply:
Yes, `requestURL` should be the right parameter name at the moment. Naming is a bit chaotic at the moment, and I still haven't put enough effort on solving that waiting for a concensus on the naming (they may not be even called "pluginIntegrations").

Reply:
Addressed on 3f8023b